source/js/main.js

```diff
@@ -64,6 +65,7 @@ let main = {
       this.injectReactComponents();
       this.bindHandlers();
       initializePrimaryNav(networkSiteURL, csrfToken, primaryNav);
+      this.initPageSpecificScript();
```

Comment:
As a generic function, we probably want to move this call to line 72, so that it runs after React is done.

Reply:
Updated!